### PR TITLE
First css commit: basic header and body margins

### PR DESCRIPTION
**Pull request by DamonRomano**

### CODE REVIEW

File: doa.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+font-weight: bold;
+font-size: 3em;
+}
+
+
+
+
+
+
+/*BODY*/
```
Comment (josephkane) [inline doa.css:46]: I think the best flow would be body, .mainContent, #splashPicture, #splashPicture h1

File: doa.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+#splashPicture img {
+  width: 100%;
+}
+
+
+body {
+  font-family: "Metal Mania", sans-serif;
+  margin-top:200px;
```
Comment (josephkane) [inline doa.css]: Not sure if this is necessary

File: doa.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+}
+#header h1 {
+  font-size: 2.5em;
+  font-weight: bold;
+}
+
+
+
+
+.navHome {
```
Comment (josephkane) [inline doa.css]: Not sure if this is necesarry

File: doa.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+ font-size: 5em;
+ font-weight: bold;
+}
+
+#navLinks {
```
Comment (josephkane) [inline doa.css]: Not sure if this is necesarry

File: doa.css (hunk 1 above)
Comment (whitney-mitchell) [inline doa.css:46]: That's what I was going to say. Largest containers first, then their contents.

**Comment by whitney-mitchell** [comment]
Other than that cascade, looks good so far.

**Comment by josephkane** [comment]
:+1:

**Pull request merged**